titus-common-ext/kube/src/main/java/com/netflix/titus/ext/kube/clustermembership/connector/transport/fabric8io/crd/FClusterMember.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.ext.kube.clustermembership.connector.transport.fabric8io.crd;
+
+import java.util.List;
+import java.util.Map;
+
+import io.fabric8.kubernetes.model.annotation.Plural;
+import io.fabric8.kubernetes.model.annotation.Singular;
+
+@Singular("clusterMember")
+@Plural("clusterMember")
```

Comment:
clusterMembers?

Also, arent CRD names always lowercase by convention? E.g.: `clustermember` and `clustermembers`?